### ISSUE TITLE
Glossary per Architecture

### ISSUE THREAD

**Issue by 00alis** [arduino/glossary-content#7]
It would be really really helpful to have a page per architecture we use:
AVR
SAM
SAMD
ARC32
I586
I686

Listing all the boards that belong to each architecture.
@simonePDA
@akash73

**Comment by SimonePDA** [comment]
@00alis 
I agree. I still have to learn ASCIIDOC and I am also trying to elaborate and propose a structure for the glossary terms. Currently the terms and texts of Create glossary are short (or very short)  and don't contain any external link.

I was thinking about a structure that is:

**Term**
_short (quick and dirty) explanation_
(read more...)
Simple, but technically sound explanation of the term/technology
**Suggested readings**
Some external links with references and in depth contents.

what do you think?

**Comment by 00alis** [comment]
reporting here @SimonePDA answer I got via email

> I agree. I still have to learn ASCIIDOC and I am also trying to elaborate and propose a structure for the glossary terms. Currently the terms and texts of Create glossary are short (or very short) and don't contain any external link.
> 
> I was thinking about a structure that is:
> 
> **Term**
> short (quick and dirty) explanation
> (read more...)
> Simple, but technically sound explanation of the term/technology
> **Suggested readings**
> Some external links with references and in depth contents.
> 
> what do you think?

It sounds good to me! We are making great progress with the Reference finally and you can see the work in progress here https://cdn.arduino.cc/reference/en/#documentation
In the side bar you can see the item 'Glossary', attached the mockups. 
[Glossary.pdf](https://github.com/arduino/glossary-content/files/231637/Glossary.pdf)

Thanks!

**Comment by 00alis** [comment]
Hi @SimonePDA, @mastrolinux created templates for you to reuse: 
https://github.com/arduino/glossary-content/blob/master/en/led.adoc is the example for a single line Glossary definition. There is a great starting point in the StarterKit Project book, it would be a matter of translating each entery in Adoc format. Let me know if you have the Starterkit Glossary in digital version, I will provide you with that otherwise.
`title` is the glossary term (e.g. LED), while `categories` is the alphabetical letter the term should be listed under (e.g. L).


Here how each single line page should be formatted:
```
---
title: "OneLine"
categories: ["O"]
---

= One Line

Just fill this line with the text you will find in the Starter Kit page.
```


For the longer definitions (e.g. BLE), here is the template:
https://github.com/arduino/glossary-content/blob/master/en/example.adoc
Here how each long-description pages should be formatted:
```
---
title: "LongDescription"
categories: ["L"]
---

= Long Description 

This is the one line definition that will show up in the landing page.<!--more-->
All the extra text is here, it will be displayed in a new page.
Example text: It consists of almost 10.454.555 lines of code that does absolutely nothing.
the Atomizer was awarded of the IGNobel price for 2 years, and many other big prizes.
MoMa asked to the original creator to show it in the famous museum but he refused .
Now the Atomizer lives in a Ubuntu Mint distro's trashcan that lays in an unused bookshelf in Hum (Italian: Colmo; German: Cholm), a town in the central part of Istria, northwest Croatia, 7 km from Roč. The elevation is 349 m. It has a population of seventeen (17) as per the 2001 census,but is officially a town.
courtesy of https://en.wikipedia.org/wiki/Hum,_Croatia[Wikipedia] (use Asciidoc syntax here)

image::http://arduino.cc/en/uploads/Main/ArduinoUno_R3_Front_450px.jpg[caption="", title="A beautiful Arduino UNO"]

Some more text that explains in detail this term.
```

For reference here is the mockup 
[Glossary.pdf](https://github.com/arduino/glossary-content/files/359139/Glossary.pdf)
